client/src/tests/e2e/PlacesList.spec.mjs

```diff
@@ -0,0 +1,60 @@
+import {test, expect} from '@playwright/test';
+import {addNewList, addNewPlace, submitCredentials} from "./Steps.mjs";
+import {config} from 'dotenv'
+config({ path: './.env.test' });
+
+let suffix
+let login
+let password
+
+const host = 'http://localhost'
+const port = 3000
+
+test.describe('Places lists', () => {
+    test.beforeEach(async ({page}) => {
+        if (!login && !password) {
+            login = 'login' + Math.random()
+            password = 'password' + Math.random()
+            await page.goto(`${process.env.E2E_TESTS_URL}/sign-up`)
```

Comment:
btw in playwright you can define playwright config with the base URL in the case you don't have to repeatably write  process.env.*

you would have just page.goto('/sign-up')

playwright.config.ts could be like:

const config: PlaywrightTestConfig = {
  use: {
    baseURL: getBaseUrl(),
    headless: true,
    viewport: { width: 1280, height: 720 },
    ignoreHTTPSErrors: true,
  },
}

where getBaseUrl could be like util function:

export const getBaseUrl = () => {
  return process.env.BASE_URL ?? default_base_url
}